index.js

```diff
@@ -0,0 +1,82 @@
+const generateId = () => (Math.random() + 1).toString(36).substring(7);
+
+function Book(title, author) {
+  this.title = title;
+  this.author = author;
+  this.id = generateId();
+}
+
+const firstBook = new Book('Crime', 'Dostoyevsky');
+const secondBook = new Book('The Devil', 'Bulgakov');
+
+let bookArray = [firstBook, secondBook];
```

Comment:
- [ ] Kindly use the [localStorage](https://github.com/microverseinc/curriculum-html-css/blob/main/local_storage.md) to store each book created and added to the `bookArray`.
This should fix the issue that all the created books go away when refresh!

- [ ] **OPTIONAL**
I can see that you're managing default books. I suggest using a new function for that, as an example: the `defaultBook()` function, then you write the code which will generate as many books as you can as default books in the app.